app/models/manageiq/providers/openshift/container_manager/orchestration_stack.rb

```diff
@@ -0,0 +1,39 @@
+class ManageIQ::Providers::Openshift::ContainerManager::OrchestrationStack < ManageIQ::Providers::ContainerManager::OrchestrationStack
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::ContainerManager"
+  belongs_to :container_template, :foreign_key => :orchestration_template_id, :class_name => "ContainerTemplate"
+
+  def self.create_stack(container_template, params, project_name)
+    new(:name                  => container_template.name,
+        :ext_management_system => container_template.ext_management_system,
+        :container_template    => container_template).tap do |stack|
+      stack.send(:create_provider_objects, raw_create_stack(container_template, params, project_name))
+    end
+  end
+
+  def self.raw_create_stack(container_template, params, project_name)
+    container_template.instantiate(params, project_name)
+  rescue => err
+    _log.error "Failed to provision from container template(#{name}), error: #{err}"
+    raise MiqException::MiqOrchestrationProvisionError, err.to_s, err.backtrace
+  end
+
+  def provider_objects
+    resources.order(:start_time)
+  end
+
+  def create_provider_objects(objects)
+    objects.collect { |object| create_provider_object(object) }
```

Comment:
You need to add them to resources. 
```
self.resources = objects.collect { |obj| create_provider_object(object) }
```

Consider better method name. Maybe `create_provider_object` => `new_resource`